cloud/filestore/config/vhost.proto

```diff
@@ -26,7 +26,7 @@ message TVhostServiceConfig
     // Endpoint storage options.
     optional string RootKeyringName = 2;
     optional string EndpointsKeyringName = 3;
-    optional bool RequireEndpointsKeyring = 4;
+    optional bool RequireEndpointsKeyring = 4 [deprecated=true];
```

Comment:
мы по-другому оформляем
мы дефолты прописываем в .cpp-части врапперов над конфигами

Reply:
Что ты имеешь ввиду? Это же proto

В нашей кодовой базе. есть два варианта :
1. пометить как deprecated, вижу что в ydb так делается
2. закомментировать поле целиком с пометкой obsolete, такого много в nbs конфигах.

Reply:
https://github.com/ydb-platform/nbs/blob/7a5fe83d4b1d340965d4cb621e36755389744470/cloud/filestore/libs/endpoint_vhost/config.cpp#L18

Reply:
Так [deprecated = true] это не дефолтное значение

`deprecated (field option): If set to true, indicates that the field is deprecated and should not be used by new code. In most languages this has no actual effect. In Java, this becomes a @Deprecated annotation. For C++, clang-tidy will generate warnings whenever deprecated fields are used. In the future, other language-specific code generators may generate deprecation annotations on the field’s accessors, which will in turn cause a warning to be emitted when compiling code which attempts to use the field. If the field is not used by anyone and you want to prevent new users from using it, consider replacing the field declaration with a [reserved](https://protobuf.dev/programming-guides/proto3/#fieldreserved) statement.`

Reply:
блин, сорян, мне показалось, что там написано default=true, а не deprecated=true

Reply:
> пометить как deprecated, вижу что в ydb так делается
> закомментировать поле целиком с пометкой obsolete, такого много в nbs конфигах.

если поле не используется, то оно obsolete, а не deprecated
так что давай сделаем, как у нас в остальных местах - obsolete

Reply:
Fixed